haiti.gemspec

```
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage      = 'https://noraj.github.io/haiti/'
   s.license       = 'MIT'
 
-  s.files         = Dir['bin/*'] + Dir['lib/**/*.rb'] + Dir['data/*.json']
+  s.files         = Dir['bin/*'] + Dir['lib/**/*.rb'] + Dir['data/*.json'] + Dir['man/**/*']
```

```suggestion
  s.files         = Dir['bin/*'] + Dir['lib/**/*.rb'] + Dir['data/*.json'] + Dir['man/haiti.1']
```

As we don't want to ship the ronn file.

We also want to add `ronn-ng` as a dev dependency in https://github.com/noraj/haiti/blob/887ef7751b345984ec61c10f67fc2c2b0c916585/Gemfile#L25-L28

---

Understood! I'll update the `s.files` variable, and add the dependency!

---

What about the `.ronn` file? Is it okay to leave it in `man/` directory or move it to some other directory?

---

> What about the `.ronn` file? Is it okay to leave it in `man/` directory or move it to some other directory?

It can stay there but that's why I asked for updating s.files without a wildcard.

---

Updated!